actors/builtin/market/market_actor.go

```diff
@@ -403,33 +403,46 @@ func (a Actor) ActivateDeals(rt Runtime, params *ActivateDealsParams) *abi.Empty
 //	DealIDs    []abi.DealID
 //	SectorType abi.RegisteredSealProof
 //}
-type ComputeDataCommitmentParams = market0.ComputeDataCommitmentParams
+//type ComputeDataCommitmentParams = market0.ComputeDataCommitmentParams
+type SectorDataCommitmentInputs struct {
+	DealIDs    []abi.DealID
+	SectorType abi.RegisteredSealProof
+}
+
+type ComputeDataCommitmentParams struct {
+	Inputs []*SectorDataCommitmentInputs
+}
 
-func (a Actor) ComputeDataCommitment(rt Runtime, params *ComputeDataCommitmentParams) *cbg.CborCid {
+type ComputeDataCommitmentReturn struct {
+	CommDs []*cbg.CborCid
```

Comment:
Can we avoid the pointer here? It was necessary when it was a bare return value, in order to match the exported method prototype, but I don't think is needed now.